docs/current/cloud/572923-get-started.md

```diff
@@ -0,0 +1,313 @@
+
+  ![Create organization](/img/current/cloud/get-started/create-organization-success.png)
+
+1. Click *Go to dashboard* to visit the Dagger Cloud dashboard, which allows you to manage your Dagger Cloud organization and account.
+
+  ![View dashboard](/img/current/cloud/get-started/connect-cloud.png)
+
+1. Click *Connect Dagger Cloud*. You will be redirected to a page containing your Dagger Cloud token. Note this token carefully, as you will need it to connect your Dagger Cloud account with your CI provider.
+
+  :::tip
+  You can view this token at any time by browsing to the *Organization Settings* -> *Organization* page of the Dagger Cloud dashboard (accessible by clicking your user profile icon in the Dagger Cloud interface), selecting your organization and navigating to the *Configuration* tab.
+  :::
+
+  ![Get token](/img/current/cloud/get-started/get-token.png)
+
+## Step 2: Connect Dagger Cloud with your CI
+
+:::info
+At the end of this step, you will have connected Dagger Cloud with your CI provider or CI tool using your Dagger Cloud token. If you have already connected Dagger Cloud with your CI provider/tool, you may skip this step.
+:::
+
+The Dagger Cloud dashboard will not display any data until you connect your Dagger Cloud account with a CI provider or CI tool. The general procedure to do this is:
+
+- Store the token as a secret with your CI provider/in your CI tool.
+- Add the secret to your CI environment as a variable named DAGGER_CLOUD_TOKEN.
+
+:::danger
+You must store this token as a secret (not plaintext) with your CI provider and reference it in your CI’s workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project.
+:::
+
+<Tabs groupId="ci">
+<TabItem value="GitHub Actions">
+
+1. Create a new secret for your GitHub repository named `DAGGER_TOKEN`, and set it to the value of the token obtained in [Step 1](#step-1-sign-up-for-dagger-cloud). Refer to the GitHub documentation [on creating repository secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository).
+
+1. Update your GitHub Actions workflow and add the secret to your `dagger run` step as an environment variable. The environment variable must be named `DAGGER_CLOUD_TOKEN` and can be referenced in the workflow using the format `DAGGER_CLOUD_TOKEN: ${{ secrets.DAGGER_TOKEN }}`. Refer to the GitHub documentation on [using secrets in a workflow](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow).
+
+1. Install the [Dagger Cloud GitHub App](https://github.com/apps/dagger-cloud). Once installed, GitHub automatically adds a new check for your GitHub pull requests, with a link to see CI status for each workflow run in Dagger Cloud.
+
+Here is a sample GitHub Actions workflow file with the Dagger Cloud integration highlighted:
```

Comment:
I have modified the samples (except Argo) to use Node instead of Go, as this fits better with the sample app. @kpenfound could you please update the Argo sample?